src/builder/meta.rs

```diff
@@ -153,6 +153,9 @@ impl ObjectMetaBuilder {
     }
 
     pub fn build(&self) -> ObjectMeta {
+        // NOTE (Techassi): Shouldn't this take self instead of &self to consume
+        // the builder and build ObjectMeta without cloning?
+
```

Comment:
I think you're right. They do the same here: https://rust-unofficial.github.io/patterns/patterns/creational/builder.html#example

Reply:
I guess we should address this in a follow-up PR. This is not really part of this PR and would make the PR even bigger than it already is.

I will create a tracking issue, where we can track all the technical debt / cruft which needs fixing.

Reply:
Agree, a small PR is a good PR